### PR TITLE
Fix go lint errors

### PR DESCRIPTION
**Pull request by FxKu**
for main code and kubectl-pg plugin

added mock client for TestTLS unit test

### CODE REVIEW

**Comment by FxKu** [comment]
:+1:

**Comment by FxKu** [comment]
:+1:

File: pkg/cluster/volumes_test.go [hunk 1]
```diff
@@ -221,10 +221,10 @@ func TestMigrateEBS(t *testing.T) {
 }
 
 type testVolume struct {
-	iops        int64
-	throughtput int64
-	size        int64
-	volType     string
+	size int64
+	//iops        int64
```
Comment (sdudoladov) [inline pkg/cluster/volumes_test.go:225]: does that really introduce dead code ?

**Comment by Jan-M** [comment]
:+1:

**Pull request merged**